README.md

```diff
@@ -32,15 +32,21 @@ requests
 | where cloud_RoleName == "community-api"
 | where name == "PUT CaseNoteController/upsertCaseNotesToDelius"
 ```
-For offenders that don't yet exist in Delius this will create a 404 and then be retried twice.
+For offenders that don't yet exist in Delius this will create a 404 which will then be ignored.
 
 ### Running against localstack
+
 Localstack has been introduced for some integration tests and it is also possible to run the application against localstack.
 
-* Checkout localstack from [GitHub](https://github.com/localstack/localstack)
-* Make sure you have docker, docker-compose and aws CLI installed
 * In the root of the localstack project, run command `docker-compose up` to bring up localstack
-* Start the Spring Boot app with profile='localstack', env var SQS_PROVIDER=localstack and SQS_EMBEDDED=false 
+* Start the Spring Boot app with profile='localstack'
 * You can now use the aws CLI to send messages to the queue
 * The queue's health status should appear at the local healthcheck: http://localhost:8082/health
 * Note that you will also need local copies of Oauth server, Case notes API and Delius API running to do anything useful
+
+### Running the tests
+
+With localstack now up and running (see previous section), run
```

Comment:
Is it worth mentioning that command here `sudo rm -rf /tmp/localstack && docker-compose down && docker-compose up`?

Reply:
well tbh I think that's only relevant for elastic search, for this project the state isn't persisted